bridge_ui/src/utils/consts.ts

```diff
@@ -655,20 +655,20 @@ export const ALEPHIUM_BRIDGE_ADDRESS =
     ? "000000000000000000000000000000000000000000000"
     : CLUSTER === "testnet"
     ? "000000000000000000000000000000000000000000000"
-    : "uPDDKWZEgtrJZfvd3i1p6xmULjQRS9mAUVeRQ6eVdP6m";
+    : "26v7DJHTsgtGar18M47KnY7f47WzKCm2oNmLsC3CgpGUb";
```

Comment:
Maybe we need to copy the deployment JSON file to the `bridge_ui` directory because sometimes we run the bridge ui via docker, I'll try it later, if it is possible I'll open another PR for this.

Reply:
good to me. No need to do it for now.